src/GeniusYield/Test/Privnet/Setup.hs

```diff
@@ -141,12 +189,8 @@ withPrivnet testnetOpts setupUser = do
             { wallets
             , poolNodes
             , testnetMagic
-            } <- cardanoTestnetDefault testnetOpts conf
+            } <- cardanoTestnet' testnetOpts conf
 
-        era <- case cardanoNodeEra testnetOpts of
```

Comment:
There's a little issue here in the sense that it's possible for the user to pass a `CardanoTestnetOpts` that is _not_ Conway bound. Which will make everything break (since we don't handle different eras anymore).

To be clear, I think the benefits of handling multiple eras are not worth the complexity - so I appreciate the removal of the era handling stuff. But it is important to point out that this is an open trap.

Reply:
I have updated haddock to indicate it 👍